modules/codefresh/Makefile

```diff
@@ -12,7 +12,7 @@ codefresh/trigger/webhook:
 	 --data-binary \
 		'{"serviceId":"$(CF_TRIGGER_SERVICE_ID)","type":"$(CF_TRIGGER_TYPE)","repoOwner":"$(CF_TRIGGER_REPO_OWNER)","branch":"$(CF_TRIGGER_BRANCH)","repoName":"$(CF_TRIGGER_REPO_NAME)"}'
 
-## Export codefresh additional envvars
+## DEPRICATED!!! Export codefresh additional envvars
```

Comment:
*deprecated

Reply:
What's it been replaced with? Can you add some context for me.

Reply:
That code useful just for slack notifications, and we move it here
https://github.com/cloudposse/build-harness/pull/117/files#diff-bb096362386f84e1c211161d79aa9ec5R38
Because we encapsulate slack notification in the targets

Reply:
Fix typo

Reply:
Add a `## deprecated` section to PR description (jermey will appreciate it) =)